troposphere/static/js/components/modals/image_version/scripts/ScriptMultiSelectAndCreate.react.js

```diff
@@ -135,6 +135,7 @@ define(function (require) {
     render: function () {
       var createShowing = this.state.showCreateForm,
         createButtonText = this.props.createButtonText,
+        // TODO: are parens necessary?
         showFormButtonText = (!createShowing) ? this.props.showButtonText : this.props.hideButtonText;
```

Comment:
LOL This file isn't being used by the instanceLuanchModal, I didn't write this but it is a good question. 

If they work, I kind of think it is easier to read with them.